package.json

```diff
@@ -14,11 +14,14 @@
   },
   "homepage": "https://github.com/adrianmcli/truffle-react#readme",
   "dependencies": {
+    "chai": "^4.1.2",
+    "chai-bignumber": "^2.0.2",
     "openzeppelin-solidity": "^1.10.0"
   },
   "devDependencies": {
     "babel-polyfill": "^6.26.0",
     "babel-preset-env": "^1.7.0",
-    "babel-register": "^6.26.0"
+    "babel-register": "^6.26.0",
+    "token-test-suite": "0.0.3"
```

Comment:
Remove`token-test-suite` since we are not using it.

Reply:
removed